docs/main.js

```diff
@@ -0,0 +1,49 @@
+document.getElementById('contactForm').addEventListener('submit', function(event) {
+    // Clear previous error messages
+    const errorMessageDiv = document.getElementById('error-message');
+    errorMessageDiv.textContent = '';
+    const inputs = document.querySelectorAll('input, textarea');
+    inputs.forEach(input => input.classList.remove('error'));
+
+    let isValid = true;
+
+    function validateForm() {
+        var x = document.forms["myForm"]["fname"].value;
```

Comment:
Here you are trying to get the form named `myForm`, but there is no such form in your HTML.